internal/format/interpolation.go

```diff
@@ -0,0 +1,76 @@
+package format
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+
+	"github.com/alecthomas/kong"
+)
+
+// interpolationRegex is the regex to find if a given string contains variables
+// for interpolation
+var interpolationRegex = regexp.MustCompile(`(\$\$)|((?:\${([[:alpha:]_][[:word:]]*))(?:=([^}]+))?})|(\$)|([^$]+)`)
+
+// interpolate interpolates the given string s with variables from vars
+// The [upstream
+// function](https://github.com/alecthomas/kong/blob/v0.8.0/interpolate.go#L22)
+// was sadly not exported, so we had to copy it.
+func interpolate(s string, vars kong.Vars) (string, error) {
```

Comment:
pretty nasty 😄 I guess the best would be to eventually get this upstream (so placeholders are interpolated automatically) so we don't need to copy this anymore.